src/Ticket.php

```diff
@@ -5096,9 +5095,9 @@ public static function showVeryShort($ID, $forcetab = '')
                                 ]
                             )
                         );
-                         echo $name;
+                         echo htmlspecialchars($name);
```

Comment:
`$name` contains HTML. This will be handled in #17872.
```suggestion
                         echo $name;
```